packages/formatter-markdown/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@hint/formatter-markdown",
+  "version": "1.0.0",
```

Comment:
I believe the first version needs to be 0.0.1 or similar so it gest bumped automatically by our release script.